src/python/WMCore/WMExceptions.py

```diff
@@ -131,6 +131,7 @@
                       8026: "NotFound (something other than a product or dictionary not found).",
                       8027: "FormatIncompatibility.",
                       8028: "FileOpenError with fallback.",
+                      8029: "RootEmbeddedFileSequence no input files specified for secondary input source.",
```

Comment:
Thong, how about we update this error message to:
`The job configuration has no input files specified for secondary input source`?

Reply:
Ok, it is changed now.